Gems/GradientSignal/Code/Source/Components/ShapeAreaFalloffGradientComponent.cpp

```diff
@@ -291,14 +299,18 @@ namespace GradientSignal
 
     void ShapeAreaFalloffGradientComponent::SetFalloffWidth(float falloffWidth)
     {
+        AZ::Aabb dirtyRegion;
```

Comment:
nit: I'd feel a bit more comfortable if this was explicitly initialized in an invalid state instead of relying on the default construction. (Repeated several times)

Reply:
In these specific cases I wasn't worried about it because it's non-branching logic that always assigns it to a value, but I'm definitely great with adding initialization to help guarantee it always has a value as the code evolves over time. Uninitialized Aabbs are no fun to track down. :(

Reply:
My main concern is how the aabbs were later used in `NotifyRegionChanged()` to do a validity check. If the default construction of aabbs ever changed so it started in a valid state (say a 1x1x1 cube), then that code would break. That's pretty unlikely to happen, but still good to be safe :)